contracts/dev/functions/FunctionsBillingRegistry.sol

```diff
@@ -291,7 +290,7 @@ contract FunctionsBillingRegistry is
     FunctionsBillingRegistryInterface.RequestBilling memory /* billing */
   ) public pure override returns (uint96) {
     // NOTE: Optionally, compute additional fee here
-    return 200_000_000_000_000_000; // 0.2 LINK
+    return 0;
```

Comment:
Does this need to be set back to 0.2 LINK?

Reply:
Yep

Reply:
```suggestion
      return 200_000_000_000_000_000; // 0.2 LINK
```